apps/etterlatte-behandling/src/main/kotlin/oppgaveGosys/GosysOppgaveService.kt

```diff
@@ -143,7 +145,7 @@ class GosysOppgaveServiceImpl(
         logger.info("Fant ${gosysOppgaver.antallTreffTotalt} oppgave(r) med tema: $temaListe")
 
         return gosysOppgaver.oppgaver
-            .map { it.tilGosysOppgave() }
+            .map { it.tilGosysOppgave(saksbehandlerInfoDao) }
```

Comment:
Burde kanskje hente _alle_ navn/identer fra databasen, cache det til map, og mappe ved behov. 

Mange av kallene mot denne servicen henter tusenvis av oppgaver, så det blir voldsom last på databasen for noe som enkelt kan unngås.

Reply:
Enig, lagt alt i en cache nå som periodisk oppdateres.